pkg/cidata/cidata.TEMPLATE.d/boot/25-guestagent-base.sh

```diff
@@ -10,6 +10,13 @@ for f in $(seq 0 $((LIMA_CIDATA_MOUNTS - 1))); do
 	mkdir -p "${mountpoint}"
 	gid=$(id -g "${LIMA_CIDATA_USER}")
 	chown "${LIMA_CIDATA_UID}:${gid}" "${mountpoint}"
+
+	mountpointopt="LIMA_CIDATA_MOUNTS_${f}_MOUNTOPTION"
+	option="$(eval echo \$"$mountpointopt")"
+	if [ "${LIMA_CIDATA_MOUNTTYPE}" = "9p" ]; then
+		tag="mount${f}"
+		mount -t 9p -o trans=virtio "${tag}" "${mountpoint}" -o version=9p2000.L,msize=131072 -o "${option}"
```

Comment:
Can we make msize configurable in the YAML?
Can be a separate PR though.

Reply:
I think it is possible to add a separate block for parameters, like the SSHFS ?

9p version
msize